scripts/install/authelia.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#
+if [[ ! -f /install/.nginx.lock ]]; then
+    echo_warn "Nginx is required for this application"
+    exit
+fi
+
+. /etc/swizzin/sources/functions/utils
+. /etc/swizzin/sources/functions/os
+
+# Get the current version using a git ls-remote tag check
+authelia_latestv="$(git ls-remote -t --refs https://github.com/authelia/authelia.git | awk '{sub("refs/tags/", "");sub("(.*)-alpha(.*)", ""); print $2 }' | awk '!/^$/' | sort -rV | head -n1)"
+# Create the download url using the version provided by authelia_latestv
+case "$(_os_arch)" in
+    "amd64") authelia_arch="amd64" ;;
+    "arm32") authelia_arch="arm32v7" ;;
+    "arm64") authelia_arch="arm64v8" ;;
```

Comment:
![](https://media.discordapp.net/attachments/702953563340537896/829111694197260298/unknown.png)
```suggestion
    "armhf") authelia_arch="arm32v7" ;;
    "arm64") authelia_arch="arm64v8" ;;
```